src/components/BookList.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+
+function BookList() {
```

Comment:
According to the project requirements:

![image](https://user-images.githubusercontent.com/57421823/157222104-bfe57039-5cdd-4925-8344-a444ce5227db.png)

Therefore:

- [ ] Please, kindly create a new component called "Book" on a file named "Book.js".  This file will be responsible for rendering each book with the title and author taken as props.

Don't worry, I'll help you.

## Explanation

We need to create a reusable component that has each book's data.

## Hints

Create the said "Book.js" component and make it take the book data as a prop.  When done, include it inside your "BookList" component as a child.

Look at this example:

### The new Book.js:

```JAVASCRIPT

import PropTypes from 'prop-types';  // Eslint won't let you add props without these prop types validations.  In order to use them, you need to install them: npm install prop-types --save

const Book = ({ title, author }) => (  // The new Book component.  Takes the data as props to render the title and author.

      <div>
        <p>{title}</p>
        <button type="button">
          {author}
        </button>
      </div>
}

// After the function component, add validations to satisfy eslint.

BookItem.propTypes = {
  title: PropTypes.string.isRequired,
  author: PropTypes.string.isRequired,
};

export default BookItem;
```

### On your BooksList.js:

```JAVASCRIPT

import Book from './Book';

const BooksList = () => {

  const booksList = [ // This is an example list of initial books.
    { id: 1, title: 'The Hobbit', author: 'J.R.R. Tolkien' },
    { id: 2, title: 'The Da Vinci Code', author: 'Dan Brown' },
    { id: 3, title: 'Pinocchio', author: 'Carlo Collodi' },
  ];

  return (
    <>

      <ul>
        {booksList.map((book) => ( // Iterate over the list of initial books.  
          <li key={book.id}>
            <Book title={book.title} author={book.author} /> // Here we are using the new imported Book component!!
          </li>
        ))}
      </ul>

    </>
  );
};

export default BooksList;
```

Now that you know how to do it, please kindly create your new "Book" component with props, and include it in your "BookList" component.  That way, we will meet this requirement 💪  


_Note: This is just an example to help you, and should not be taken as an "official way" to do it. This is just the result of my own personal effort and experience._

Reply:
Great job @Meltrust !!